lib/simp/cli/commands/version.rb

```diff
@@ -0,0 +1,45 @@
+require 'simp/cli/commands/command'
+
+class Simp::Cli::Commands::Version < Simp::Cli::Commands::Command
+
+  def description
+    'Display the current version of SIMP.'
+  end
+
+  def help
+    puts "\n#{description}\n\nUsage:  simp version\n"
+  end
+
+  def run(args)
+    parse_command_line(args)
+    return if @help_requested
+
+    cmd = 'rpm -q simp'
+    begin
+      `#{cmd}`.split(/\n/).last.match(/([0-9]+\.[0-9]+\.?[0-9]*)/)[1]
+    rescue
+      #TODO Send this message to stderr instead of stdout?
```

Comment:
I realize this TODO comment existed prior to this patch, but since it's in the diff:

Output from `COMMAND --help` [should print to `stdout`.](https://www.gnu.org/prep/standards/html_node/_002d_002dhelp.html)  Users can easily pipe the output to other commands, like:

```sh
simp bootstrap --help | less
```